kroxylicious-operator/src/main/java/io/kroxylicious/kubernetes/operator/MetricsService.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.kroxylicious.kubernetes.operator;
+
+import java.util.Optional;
+
+import io.fabric8.kubernetes.api.model.IntOrString;
+import io.fabric8.kubernetes.api.model.Service;
+import io.fabric8.kubernetes.api.model.ServiceBuilder;
+import io.javaoperatorsdk.operator.api.reconciler.Context;
+import io.javaoperatorsdk.operator.processing.dependent.kubernetes.CRUDKubernetesDependentResource;
+import io.javaoperatorsdk.operator.processing.dependent.kubernetes.KubernetesDependent;
+
+import io.kroxylicious.kubernetes.api.v1alpha1.KafkaProxy;
+
+/**
+ * The Kube {@code Service} for exposing the proxy metrics.
+ * This is named like {@code metrics-${KafkaProxy.metadata.name}}.
+ */
+@KubernetesDependent
+public class MetricsService
```

Comment:
Does a metrics service make sense? We don't cluster metrics so I'm not following why we would define a single service to provide them, don't we just want to expose the metrics port on each container and let the cluster scrape each pod in turn? 

Or am I totally off base?

Reply:
I'm gonna remove this.